content/content/strings.md

```diff
@@ -2,6 +2,17 @@
 title: Strings
 ---
 # Strings
+In Nim, the string represents a sequence of characters. It’s a list of characters terminated by '\0'. There usage is very similar to python
+
+``` nimrod
```

Comment:
You should also follow the style found elsewhere on the page, where the output is placed in a separate code block from the code.